game.py

```diff
@@ -76,6 +88,16 @@ def turn(self):
         self.current_player.turn_started = datetime.now()
         self.choosing_color = False
 
+    def _first_card_(self):
+        # In case that the player did not select a game mode
+        if not self.deck.cards:
+            self.set_mode("classic")
+
+        while not self.last_card or self.last_card.special:
```

Comment:
The reason I had the `self.deck = Deck()` in this loop was because you potentially remove special cards from the deck completely. You could add something like `if self.last_card.special: self.deck.dismiss(self.last_card)`

Reply:
I don't think I get the potential to remove special cards from the deck with this code, can you explain the problem a bit more?

Reply:
This is a loop that draws cards until a condition is met. What happens to the cards that are drawn?